mmpose/apis/inference_3d.py

```diff
@@ -214,6 +215,103 @@ def vis_3d_pose_result(model,
         pose_limb_color = palette[[
             0, 0, 0, 16, 16, 16, 9, 9, 9, 9, 16, 16, 16, 0, 0, 0
         ]]
+    elif dataset == 'InterHand3DDataset':
+        skeleton = [[1, 2], [2, 3], [3, 4], [4, 21], [5, 6], [6, 7], [7, 8],
+                    [8, 21], [9, 10], [10, 11], [11, 12], [12, 21], [13, 14],
+                    [14, 15], [15, 16], [16, 21], [17, 18], [18, 19], [19, 20],
+                    [20, 21], [22, 23], [23, 24], [24, 25], [25, 42], [26, 27],
+                    [27, 28], [28, 29], [29, 42], [30, 31], [31, 32], [32, 33],
+                    [33, 42], [34, 35], [35, 36], [36, 37], [37, 42], [38, 39],
+                    [39, 40], [40, 41], [41, 42]]
+
+        pose_kpt_color = [
+            [14, 128, 250],
+            [14, 128, 250],
+            [14, 128, 250],
+            [14, 128, 250],
+            [80, 127, 255],
+            [80, 127, 255],
+            [80, 127, 255],
+            [80, 127, 255],
+            [71, 99, 255],
+            [71, 99, 255],
+            [71, 99, 255],
+            [71, 99, 255],
+            [0, 36, 255],
+            [0, 36, 255],
+            [0, 36, 255],
+            [0, 36, 255],
+            [0, 0, 230],
+            [0, 0, 230],
+            [0, 0, 230],
+            [0, 0, 230],
+            [0, 0, 139],
+            [237, 149, 100],
+            [237, 149, 100],
+            [237, 149, 100],
+            [237, 149, 100],
+            [230, 128, 77],
+            [230, 128, 77],
+            [230, 128, 77],
+            [230, 128, 77],
+            [255, 144, 30],
+            [255, 144, 30],
+            [255, 144, 30],
+            [255, 144, 30],
+            [153, 51, 0],
+            [153, 51, 0],
+            [153, 51, 0],
+            [153, 51, 0],
+            [255, 51, 13],
+            [255, 51, 13],
+            [255, 51, 13],
+            [255, 51, 13],
+            [103, 37, 8],
```

Comment:
Consider removing the last comma so multiple elements can be placed in one line.

Reply:
Fixed